project_event/models/project_task.py

```diff
@@ -175,6 +176,28 @@ def _onchange_resource_type(self):
         self.room_id = None
         self.equipment_id = None
 
+    @api.onchange('room_id')
+    def verify_room_bookable(self):
+        if self.room_id:
+            if not self.room_id.is_bookable:
+                raise ValidationError(
+                        _(
+                            'This room is not bookable'
+                        )
+                    )
+                self.room_id = None
+
+    @api.onchange('equipment_id')
+    def verify_equipment_bookable(self):
+        if self.equipment_id:
```

Comment:
@eilstWhat is the validation message "if self.equipment_id: " condition fails Or if this condition is going to true always what is the use of checking this condition.